account_invoice_partner_no_autofollow/views/res_config_settings.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.account</field>
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="40" />
+        <field name="inherit_id" ref="base.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='analytic']" position="after">
+                <h2>Auto follow</h2>
+                <div class="row mt16 o_settings_container" id="not_autofollow_setting">
+                    <div class="col-12 col-lg-6 o_setting_box">
+                        <div class="o_setting_left_pane">
+                            <field name="no_auto_follow" />
+                        </div>
+                        <div class="o_setting_right_pane">
+                            <label for="no_auto_follow" />
+                            <div
+                                class="text-muted"
+                            >Partner is not added as follower</div>
```

Comment:
**Invoice** partner is not added as follower

Reply:
done